winlink_monitor.py

```diff
@@ -150,15 +172,21 @@ def load_config(config_file):
         sys.stderr.write(f"ERROR: Could not find model {model_str} in Hamlib. See documentation for help.\n")
         sys.exit(1)
 
-    
+
     # Nodes
-    # TODO: This is tricky - add better error messaging 
+    # TODO: This is tricky - add better error messaging
     global NODES
     for nodeobj in config.get("nodes", []):
         node = Node(nodeobj["name"], nodeobj["frequency"], nodeobj["peer"])
+        if args.nodes:
+            if node.name not in args.nodes and node.peer not in args.nodes:
+                continue
```

Comment:
Should we throw an error for an invalid config? I generally prefer fail fast over silently drop.

Reply:
I am not checking the config.  I am seeing if the user supplied node identification matches something in our list.  The error would be if the user did not supply a node identification that was in our list.  They could easily typo the command line and this will take the safe route and do nothing.